product_docs/docs/odbc_connector/12.2.0.2/index.mdx

```diff
@@ -23,6 +23,6 @@ This guide contains installation information for the EDB ODBC as well as informa
 
 <div class="toctree" maxdepth="3">
 
-whats_new requirements_overview edb-odbc_overview creating_a_data_source edb-odbc_connection_properties edb-odbc_driver_functionality scram_compatibility conclusion
+odbc_release_notes requirements_overview edb-odbc_overview creating_a_data_source edb-odbc_connection_properties edb-odbc_driver_functionality scram_compatibility conclusion
```

Comment:
I don't think the toctree is used in the current framework and can be removed. It also does no harm so it isn't critically we remove them.